harvard-university-for-the-creative-arts.csl

```diff
@@ -223,19 +260,30 @@
         <text variable="number" suffix="."/>
       </else-if>
       <else-if type="motion_picture broadcast">
-        <text variable="medium" prefix="[" suffix="] "/>
+        <choose>
+          <if match="any" variable="collection-title container-title">
+            <text term="in" text-case="capitalize-first" suffix=": "/>
+            <text variable="collection-title" font-style="italic" suffix="."/>
+            <text variable="container-title" font-style="italic" suffix="."/>
```

Comment:
@ibadger, do you know if there are cases in which the items may have both titles (collection and container)? Would you get correct output in that case?